draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -2737,24 +2795,26 @@ Internet-Draft                 ACTN POI                       July 2023
    being able to use WSON and Flexi-grid topology models together
    (through multi-inheritance) is not only useful in cases of mixed
    fixed-grid and flexible-grid DWDM network topology but also the only
-   viable option in case of a mixed CWDM and flexible-grid DWDM network
-   topology.
-
-   Although not analysed in this document, it has been noted that the TE
-   Tunnel model, defined in [TE-TUNNEL], needs also to be enhanced to
-   support scenarios where multiple parallel TE paths are used in load-
-   balancing to carry the traffic between two end-points (e.g., VPN
-   traffic between two PEs).
+   viable option in case of a mixed CWDM and DWDM network topology.
 
+   Although not applicable to this document, it has been noted that the
+   WDM tunnel model would support also optical tunnel setup in case of a
+   mixed CWDM and DWDM network topology.
 
 
 
 
-Peruzzini et al.       Expires January 7, 2024                [Page 51]
+Peruzzini et al.        Expires July 30, 2024                 [Page 52]
 
-Internet-Draft                 ACTN POI                       July 2023
+Internet-Draft                 ACTN POI                    January 2024
 
 
+   Although not analysed in this document, it has been noted that the TE
+   Tunnel model, defined in [TE-TUNNEL], needs also to be enhanced to
+   support scenarios where multiple parallel TE paths are used in load-
+   balancing to carry the traffic between two end-points (e.g., VPN
+   traffic between two PEs).
+
 7. Security Considerations
 
    This document highlights how the ACTN architecture can deploy packet
```

Comment:
Should we add some considerations about LLDP security? Could Daniel King or Brent have some proposal on this topic?

Reply:
yes: see issue #40